packages/backend/src/managers/catalogManager.ts

```diff
@@ -0,0 +1,53 @@
+import type { Catalog } from '@shared/src/models/ICatalog';
+import path from 'node:path';
+import fs from 'node:fs';
+import defaultCatalog from '../ai.json';
+import type { Category } from '@shared/src/models/ICategory';
+import type { Recipe } from '@shared/src/models/IRecipe';
+import type { ModelInfo } from '@shared/src/models/IModelInfo';
+
+export class CatalogManager {
+  private catalog: Catalog;
```

Comment:
to make it really private you could use #

```suggestion
  #catalog: Catalog;
```

Reply:
Yeah but for readability, user not knowing what # mean, will not think it is priovate, can we use both ?
```
private #catalog: Catalog
```

Reply:
no you can't

Reply:
Gonna keep private for now as we are not yet using the #